runtime/altair/src/lib.rs

```diff
@@ -1245,8 +1246,26 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
+	upgrade::Upgrade,
 >;
 
+/// Runtime upgrade logic
+mod upgrade {
+	use super::*;
+	use frame_support::{traits::OnRuntimeUpgrade, weights::Weight};
+
+	pub struct Upgrade;
+	impl OnRuntimeUpgrade for Upgrade {
+		fn on_runtime_upgrade() -> Weight {
+			let mut weight = 0;
+			weight += InterestAccrual::upgrade_to_v1();
+			weight += Loans::reference_active_rates();
+			weight += InterestAccrual::remove_unused_rates();
+			weight
```

Comment:
Since we start at `0`, we could do: 
```
InterestAccrual::upgrade_to_v1() + Loans::reference_active_rates() + InterestAccrual::remove_unused_rates()
```

 in the body of `on_runtime_upgrade` ?

Reply:
We could, but I wanted the structure to emphasize that we're doing a sequence of actions (that each just happen to return a weight).